### DISCUSSION TITLE
Specialized Pathways to DataCap

### DISCUSSION THREAD

**Discussion by kevzak** [filecoin-project/notary-governance#848] [Ideas]
## Different application pathways by:

- Data Type
    - OPEN PUBLIC
    - COMMERCIAL PUBLIC
    - PRIVATE
  
- Applicant Type
    - Data Owner
    - Data Preparer
    - Lead SP / Onboarding SP
    - Aggregator / ISV / Proxy Client

## Why

- Separating pathways allows more specific requirements per pathway
- Ability to provide different views for each pathway
- Also to potentially allows for different notary views when looking to review applications they can choose by data type in filplus.fil.storage

## How

- All applications go through filplus.storage and on the application we add the Data Type and Applicant Type drop downs with specific  requirements
    - Example: As we've learned via the E-Fil+ pilot, all PRIVATE datasset applications go through required upfront client/data/SP checks

**Comment by orvn** [comment]
This makes a lot of sense to me and is a much better user segmentation than just doing things blindly by size.

It has the potential to unify all DataCap applications into a single repo instead of splitting across the [`filecoin-project/filecoin-plus-client-onboarding`](https://github.com/filecoin-project/filecoin-plus-client-onboarding) and [`filecoin-project/filecoin-plus-large-datasets`](https://github.com/filecoin-project/filecoin-plus-large-datasets) repos, if that's something we want (it does feel cleaner).

Segmentation can be done via Github labels as we do now.

> _All applications go through filplus.storage and on the application we add the Data Type and Applicant Type drop downs with specific requirements_

This sounds good. Although we'll probably want to also modify the issue templates to include these fields for direct Github applicants too.

**Comment by kevzak** [comment]
Thanks for comments, moved this to a proposal: https://github.com/filecoin-project/notary-governance/issues/872
